stdlib/2.7/getopt.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import List, Tuple
+
+class GetoptError(Exception):
+    opt = ... # type: str
+    msg = ... # type: str
+    def __init__(self, msg: str, opt: str=...) -> None: ...
+    def __str__(self) -> str: ...
+
+error = GetoptError
+
+def getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str, str]],
+                                             List[str]]: ...
+
+def gnu_getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str,str]],
+                                             List[str]]: ...
+
+def do_longs(opts: List[Tuple[str,str]], opt: str,
```

Comment:
I don't know, I just added everything that was in the source. Those functions are basically only helper functions for the "real" functions of that module, but they are not marked as private. I have to admit, I rarely have a look into the documentation due to it's low quality. Do you only want documented functions or do you want all non-private functions?